src/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -101,7 +101,6 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
 		                if (afflOld.Primary__c
 								&& !affl.Primary__c
 								&& lookupFieldName != null
-								&& UTIL_Describe.isValidField('Contact', lookupFieldName)
```

Comment:
@xinnanlisf I'm curious why this is failing for namespace builds, but not managed package builds. Are we passing a fieldname as a string, so it's not getting the namespace added correctly?

Reply:
@lparrott great question! It looks like Util_Describe.isValidField is failing:

`    public static boolean isValidField(String objectName, String fieldName) {
        // fields in our own package must not have their prefix for the Describe Field Map
       fieldName = UTIL_Namespace.StrTokenRemoveNSPrefix(fieldName);

        // make sure we have this field's schema mapped
            if (!fieldDescribes.containsKey(objectName) || !fieldDescribes.get(objectName).containsKey(fieldName))
                fillFieldMapsForObject(objectName, fieldName);

            Schema.DescribeFieldResult dfr = fieldDescribes.get(objectName).get(fieldName);
            return dfr != null;
        
    }`

If we remove fieldName = UTIL_Namespace.StrTokenRemoveNSPrefix(fieldName);, that unit test would pass. It sounds like namespace scratch org does not think primary__household__c is a valid field. Rather, it thinks hed__primary__household__c is valid.

Reply:
Gotcha, thanks for looking into it! We can chalk that up to just a quirk of namespace scratch orgs (which should never exist anyways)